reliquery/relic.py

```diff
@@ -58,6 +56,11 @@ def __init__(
             self._ensure_exists()
 
         self.metadata_db = MetadataDB()
+        self.relic = self.metadata_db.sync_relic_data(
+            RelicData(self.name, self.relic_type, self.storage_name)
+        )
+        if not self.relic:
```

Comment:
probably makees more sense to let the function that you call raise the exception as noted in the other comments.

Reply:
iim  ok if its only in the new code where you make the changes. we can follow up with another pr.

just to be clear exception handling can be good. The situation where we dont want it is if we are leaving the program in a confused state afterwards. Catching the exception should mean that things are consistent afterwards. For example if we created a tag and we catch the exception and now the tag is not in the in memory db, then this would be "bad." We may as well let teh exception bubble up because the program is in a state that can lead to mistakes. If we catch it because we can handle it and it means something (for instance when you catch that there is no tags), then this is ok because you have left things clean.